.github/workflows/pytorch112_aarch64.yml

```diff
@@ -0,0 +1,73 @@
+name: pytorch1_12-aarch64
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    - cron: '0 17 * * *' # 1:00 am UTC+8:00
+  workflow_dispatch:
+
+jobs:
+  build:
+    if: github.repository == 'alibaba/BladeDISC'
+    # The type of runner that the job will run on
+    runs-on: [self-hosted, aarch64]
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.4.0
+    - name: Build Dev Docker
+      run: |
+        set -e
+        git submodule sync
+        git submodule update --depth=1 --init --recursive
+        cp /etc/apt/sources.list .
+        docker build -t disc-dev-cpu-aarch64 \
+          --build-arg BASEIMAGE=ubuntu:20.04 \
+          --build-arg DISC_HOST_TF_VERSION="tensorflow-aarch64==2.8" \
+          -f docker/dev/Dockerfile.aarch64 .
+    - name: Build And Test DISC
+      run: |
+        set -e
+        docker run --rm -t --user $(id -u):$(id -g) \
+          -v /mnt/cache:$HOME/.cache \
+          -v /etc/passwd:/etc/passwd:ro \
+          -v /etc/group:/etc/group:ro \
+          -v $PWD:/disc \
+          -e GITHUB_WORKFLOW=$GITHUB_WORKFLOW \
+          -e TORCH_BLADE_BUILD_WITH_CUDA_SUPPORT=OFF \
+          -e TORCH_BLADE_CI_BUILD_TORCH_VERSION=1.12.0+aarch64 \
+          -w /disc \
+          disc-dev-cpu-aarch64 \
+          bash ./scripts/ci/test_pytorch_blade.sh
+    - name: Deploy Dev Docker Image
+      if: github.event.ref == 'refs/heads/main' 
+      env:
+        ALIYUN_DOCKER_USERNAME: ${{ secrets.ALIYUN_DOCKER_USERNAME }}
+        ALIYUN_DOCKER_PASSWORD: ${{ secrets.ALIYUN_DOCKER_PASSWORD }}
+        DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+        DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+        GITHUB_PULL_REQUEST: ${{ github.event.number }}
+        LOCAL_DEV_DOCKER: disc-dev-cpu-aarch64
+        REMOTE_DEV_DOCKER: bladedisc:latest-devel-cpu-aarch64
+      run: |
+        set -e
+        echo "Try to deploy runtime docker image..."
+        source $HOME/.cache/proxy_config
+        bash ./scripts/ci/deploy_tf_wrapper.sh
```

Comment:
`deploy_tf_wrapper.sh` ?

Reply:
https://github.com/alibaba/BladeDISC/pull/474/files
From above commit, it seems that we use `deploy_tf_wrapper.sh` both in `pytorch` and `tensorflow` at the very beginning. Also, pytorch110_aarch64.yaml use deploy_tf_wrapper.sh.